source/mainnet/conf.py

```diff
@@ -162,6 +162,7 @@
     "page_sidebar_items": ["page-toc"],
     "footer_items": ["footer"],
     "show_prev_next": True,
+    "navbar_align": "left",
```

Comment:
What does this change?

Reply:
The alignment of the menu bar across the top of the site. After adding Use ID it became too long.